bftengine/src/bftengine/ControlStateManager.cpp

```diff
@@ -14,12 +14,34 @@
 #include "ControlStateManager.hpp"
 namespace bftEngine {
 
-void ControlStateManager::setStopAtNextCheckpoint() {}
+// This method will be called from the command handler. Notice that the command handler does not aware to the number of
```

Comment:
"does not aware to" -> "is not aware of"

Reply:
fixed